ros/lib/host_inventory.py

```diff
@@ -0,0 +1,10 @@
+import requests
+import json
+
+
+def fetch_host_from_inventory(insights_id, rh_identity):
+    host_api_url = f"http://localhost:8081/api/inventory/v1/hosts?insights_id={insights_id}"
```

Comment:
Worth extracting an inventory URL in a environment variable as URL will change as per environment (dev/qa/prod)?